apps/mobile/src/app/(home)/account/[account]/account.layout.tsx

```diff
@@ -1,35 +1,35 @@
 import { getAvatarIcon } from '@/components/avatar-icon';
 import { Balance } from '@/components/balance/balance';
 import { PageLayout } from '@/components/page/page.layout';
-import {
-  CollectiblesWidget,
-  mockCollectibles,
-  serializeCollectibles,
-} from '@/components/widgets/collectibles';
-import { TokensWidget } from '@/components/widgets/tokens';
-import { MockedAccount } from '@/mocks/account.mocks';
-import { mockTotalBalance } from '@/mocks/balance.mocks';
-import { getMockTokens } from '@/mocks/tokens.mocks';
+import { TokensWidget } from '@/components/widgets/tokens/tokens-widget';
+import { Token } from '@/components/widgets/tokens/types';
+import { Account } from '@/store/accounts/accounts';
+
+import { Money } from '@leather.io/models';
 
 import { AccountOverview } from './account-overview-card';
 
 interface AccountLayoutProps {
-  account: MockedAccount;
+  account: Account;
+  balance: Money;
+  tokens: Token[];
 }
 
-export function AccountLayout({ account }: AccountLayoutProps) {
+export function AccountLayout({ account, balance, tokens }: AccountLayoutProps) {
   return (
     <PageLayout>
       <AccountOverview
         Icon={getAvatarIcon(account.icon)}
-        heading={<Balance balance={mockTotalBalance} variant="heading02" />}
+        heading={<Balance balance={balance} variant="heading02" />}
         caption={account.name}
       />
-      <TokensWidget tokens={getMockTokens()} totalBalance={mockTotalBalance} />
+      <TokensWidget tokens={tokens} totalBalance={balance} />
+      {/* TODO - re-enable widgets when real data available */}
```

Comment:
@fbwoolf - if you want to help and start working on this, coming soon is [adding stacks NFTs](https://linear.app/leather-io/issue/LEA-1526/load-account-level-stacks-nfts)

We will need to load them with a query and pass them through `serializeCollectibles`

Reply:
Think I'm assigned to work on push notifications atm. I'll jump in here if that work gets put on hold.